deploy/starfleet-execution-role-org-management.yml

```diff
@@ -0,0 +1,223 @@
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Required Parameters for Starfleet Execution Role
+        Parameters:
+          - AdministrativeRoleName
+          - OrganizationId
+          - StarfleetAccountId
+      - Label:
+          default: Optional Parameters for Starfleet Execution Role
+        Parameters:
+          - AdministrativeRolePath
+          - ExecutionRoleName
+          - ExecutionRolePrefix
+          - Partition
+
+Parameters:
+  AdministrativeRoleName:
+    Type: String
+    Description: >-
+      (Required) The friendly name (not ARN) of an IAM role that will be authorized to assume
+      the Starfleet execution roles in every managed account. This role will be used by
+      Starfleet administrators for local development and executing the Starfleet CLI.
+    AllowedPattern: '^[\w+=,.@*-]{1,64}$'
+    MinLength: 1
+    MaxLength: 64
+  OrganizationId:
+    Type: String
+    Description: >-
+      (Required) The unique identifier (ID) of the AWS Organization in which Starfleet
+      is being deployed. Starfleet execution roles will be configured to deny any attempt
+      to assume the role from an external IAM principal.
+    AllowedPattern: '^o-[a-z0-9]{10,32}$'
+    MinLength: 10
+    MaxLength: 32
+  StarfleetAccountId:
+    Type: String
+    Description: >-
+      (Required) The 12-digit AWS account ID where Starfleet is deployed and managed.
+    AllowedPattern: '\d{12}'
+    MinLength: 12
+    MaxLength: 12
+
+  AccountIndexRoleName:
+    Type: String
+    Description: >-
+      (Optional) The name of the Starfleet IAM execution role that will be created
+      for the Starfleet Account Index Generator.
+    Default: starfleet-account-index-execution-role
+    AllowedPattern: '^[\w+=,.@*-]{1,64}$'
+    MinLength: 1
+    MaxLength: 64
+  AdministrativeRolePath:
+    Type: String
+    Description: >-
+      (Optional) The path to the Starfleet administrative IAM role.
+    Default: /
+    AllowedPattern: '(\u002F)|(\u002F[\u0021-\u007E]+\u002F)'
+    MinLength: 1
+    MaxLength: 512
+  ExecutionRoleName:
+    Type: String
+    Description: >-
+      (Optional) The name of the Starfleet IAM execution role that will be created.
+    Default: starfleet-worker-execution-role
+    AllowedPattern: '^[\w+=,.@*-]{1,64}$'
+    MinLength: 1
+    MaxLength: 64
+  ExecutionRolePrefix:
+    Type: String
+    Description: >-
+      (Optional) The name prefix that is used across all Starfleet execution roles. The role
+      prefix is used to restrict which roles in the Starfleet account can assume Starfleet
+      execution roles.
+    Default: starfleet-*
+    AllowedPattern: '^[\w+=,.@*-]{1,32}$'
+    MinLength: 1
+    MaxLength: 32
+  Partition:
+    Type: String
+    Description: >-
+      (Optional) The AWS partition name in which Starfleet resources are being deployed.
+    Default: aws
+    AllowedValues:
+      - aws
+      - aws-cn
```

Comment:
One issue right now is that starfleet doesn't have any logic for handling cn or gov-cloud partitions.

Reply:
Fair shout, fixed in 5c37a219b5939d4e16f41939d923cfc393198711.